xml_struct/src/lib.rs

```diff
@@ -134,4 +134,10 @@ pub trait XmlSerializeAttr {
 pub enum Error {
     #[error("failed to process XML document")]
     Xml(#[from] quick_xml::Error),
+
+    /// An error representing a failure in formatting a data structure prior to
+    /// serializing it into XML. Its inner type is generic on purpose, as the
+    /// specific error type might be defined by a third-party crate.
+    #[error("unexpected error when formatting data structure prior to serialization")]
+    Format(#[from] anyhow::Error),
```

Comment:
Formatting doesn't seem like quite the right term here; this could be any sort of error in serializing a type. I could see this being used for something like a string type in a non-UTF-8 encoding which needs to be transcoded. How do you feel about `Value`, since it represents an error in serializing a specific value?

Reply:
That's a much better name, thanks. I wasn't the biggest fan of `Format` but failed to find something better.